lib/all_members_page.rb

```diff
@@ -4,16 +4,9 @@
 class AllMembersPage < Scraped::HTML
   decorator Scraped::Response::Decorator::AbsoluteUrls
 
-  field :members do
+  field :member_urls do
     noko.xpath('id("lists_list_elements_35")//tr[td]').map do |tr|
-      tds = tr.css('td')
-      {
-        name:    tds[1].text.tidy,
-        faction: tds[2].text.sub('Фракция ', '').tidy,
-        area:    tds[4].text.tidy,
-        image:   tds[0].css('img/@src').text,
-        source:  tds[1].css('a/@href').text,
-      }
+      tr.css('td:first a/@href').text
```

Comment:
Oops, yep, changed in https://github.com/everypolitician-scrapers/russia-duma-2016/pull/3/commits/aa960ffc970893835d719c2d78a5d74b39584cfb.